src/GraphQL.php

```diff
@@ -144,6 +145,31 @@ public static function buildSchema($source, $resolverRegistry, array $options =
             );
     }
 
+    /**
+     * @param SchemaInterface                 $schema
+     * @param string|Source                   $source
+     * @param array|ResolverRegistryInterface $resolverRegistry
```

Comment:
Should this really be `array` or should it be `callable`?

Reply:
It is correct, that's if you want to provide the resolver registry as an array (it will be instantiated as a real `ResolverRegistry` inside the `GraphQL` class.